### ISSUE TITLE
crash after crafting skill transfer

### ISSUE THREAD

**Issue by myrialeen** [LegionDark/Issues#1717]
@Heepster03 
Date & Time:
11 december 2017

FFXI Client Version (use /ver):
30171103_1

Server's Expected Client Version matches, yes/no? (use !ecv):
3017110

Character Name:
Nimueh

Nation:
Bastok

Job(level)/Sub Job(level):
THF99 all other 75 after the transfer

NPC or Monster or item Name:
NA

Zone name:
NA

Coordinates (use !where):
NA

ffxiah.com link (for items issues only):
NA

Multi-boxing? (multiple clients on same connection):
NA

Steps To Reproduce / any other info:

Yesterday a gm transferred all my crafting skill but after few minutes i crashed and i lost some one

[My Skill](https://image.ibb.co/ktNrvG/img_20171210_163832.png)
Thank you

**Comment by Heepster03** [comment]
Yes, I transferred them. What did you lose? It should've saved to the database.

**Comment by myrialeen** [comment]
Woodworking had 66 now i have 6
Smithing had 18 now i have 1
Goldsmithing had 24 now i have 2
Clothcraft and Leather craft are 0  and had 2-1
Alchemy had 23 now 2
Cooking had 10 now 1
(seems i lost the 2nd number..)

**Comment by Jessland86** [comment]
This is caused by DSP breaking something with a GM command as referenced in this ticket. https://github.com/LegionDark/Issues/issues/1769 Once this is fixed we'll be able to properly get your skills set.
